Feature/f584e54b

bh2980 opened the pull request:
Company layout 검토 부탁드립니다.

bh2980 committed:
feat(Company) : rough layout

bh2980 committed:
Merge branch 'feature/f584e54b' into develop

bh2980 committed:
feat(Company) : Rough layout 2

not yet em, rem unit

bh2980 committed:
feat(Company) : almost completed layout

bh2980 committed:
feat(Company) : layout polished

bh2980 committed:
Merge remote-tracking branch 'origin/develop' into develop

bh2980 committed:
Merge branch 'develop' into feature/f584e54b

bh2980 committed:
Merge branch 'develop' into feature/f584e54b

kasterra commented:
![fix](https://user-images.githubusercontent.com/74085452/150294794-da25045c-b087-4b87-ae38-016d0c17a37c.png)
이거 고쳐주세요

kasterra commented:
아 그리고 아이콘은 assets/icons 폴더 안에 넣어주세요

bh2980 committed:
fix(Company) : correct li

Climier-code commented on src/components/Company/Company.jsx:
```diff
@@ -0,0 +1,40 @@
+
+const Company = () => {
+  return (
+    <>
```
불필요한 Fragment 사용

bh2980 replied:
아 강의에서 쓰길래 써야하는 건 줄 알았어요.
Fragment라는걸 이걸 보고 알았읍니다...
반영 완료!

Climier-code commented on src/components/Company/Company.jsx:
```diff
@@ -7,27 +7,86 @@ const Company = () => {
+              </div>
+              <div className="company-name">Apple Inc. (APPL)</div>
+              <div>
+                <div className="price">$ 177.27</div>
```
**Intl.NumberFormat** **Number.toLocaleString**
각 나라 통화 별 표기 포매팅 함수들인데 참고하면 좋아용

bh2980 replied:
오 감사합니당 써볼게요!

Climier-code commented on src/components/Company/Company.jsx:
```diff
@@ -7,27 +7,86 @@ const Company = () => {
-                conasndklasknlakwqkfbladkfjasknfjadhgaiuelmadm sdalskflabelkjfbl
+            <div className="logo">
+              <div className="shadow-box company-logo">
+                <img src="https://upload.wikimedia.org/wikipedia/commons/a/ab/Apple-logo.png"></img>
```
children이 없는 태그 (특히 img, br 등등) 은 빈태그 하나로 처리하시는게 좋아요
`<img />`

bh2980 replied:
넵 반영 완료했습니다.

Climier-code commented on src/components/Company/Company.jsx:
```diff
@@ -7,27 +7,86 @@ const Company = () => {
-                conasndklasknlakwqkfbladkfjasknfjadhgaiuelmadm sdalskflabelkjfbl
+            <div className="logo">
+              <div className="shadow-box company-logo">
+                <img src="https://upload.wikimedia.org/wikipedia/commons/a/ab/Apple-logo.png"></img>
```
img 태그엔 alt는 필수입니당 아니면 eslint에서 경고띄웁니당

`<img alt="" src="" />`

bh2980 replied:
반영 완료했습니당.

Climier-code commented on src/components/Company/Company.scss:
```diff
@@ -7,6 +7,7 @@
 .wrapper {
   height: 100%;
   display: flex;
+  font-size: 2vh;
```
font-size를 정할 때는 px, em, vmin, vh 등등에서 1,2개로만 통일하는 것이 코드 정리하기에도 좋아욧

bh2980 replied:
노력했습니다.....ㅎ.....
코드 리뷰 감사합니다!

bh2980 committed:
chore(Company) : icoon file name change

bh2980 committed:
Merge remote-tracking branch 'origin/develop' into develop

bh2980 committed:
Merge branch 'develop' into feature/f584e54b

bh2980 committed:
Revert "chore(Company) : code review reflection"

This reverts commit 217aff0310832bb885894bd0bf50e118d6597492.

bh2980 committed:
fix(Company) : pull request reflection

reupload by git error
- asset/icons : folder add
- company-title-icon.svg : rename file name
- Company.jsx : delete unnecessary fragment
- Company.jsx : add alt att to img tag
- Company.jsx : edit no child tag to one tag
- Company.jsx : use Intl.NumberFormat (cuz just want to use it once)
- Company.scss : edit font-size unit to vh and em

bh2980 committed:
Merge remote-tracking branch 'origin/feature/f584e54b' into feature/f584e54b

bh2980 commented:
수정했습니다. 확인 부탁드립니다.

- asset/icons : folder add
- company-title-icon.svg : rename file name
- Company.jsx : delete unnecessary fragment
- Company.jsx : add alt att to img tag
- Company.jsx : edit no child tag to one tag
- Company.jsx : use Intl.NumberFormat (cuz just want to use it once)
- Company.scss : edit font-size unit to vh and em
- core.scss : develop branch reflection (li tag error)

bh2980 committed:
Merge remote-tracking branch 'origin/develop' into develop

bh2980 committed:
Merge branch 'develop' into feature/f584e54b

bh2980 committed:
fix(Company) : fix unloading error

Company.jsx : .format(number) -> .format(177.23)

bh2980 commented:
로딩 안되는 문제 수정했습니다. 사유는 저의 멍청....

Company.jsx : .format(number) -> .format(177.23)
